cfpb/consumerfinance.gov
# Update Cypress to 9.2.0

## Changes

- Update Cypress to 9.2.0

## How to test this PR

1. PR checks should pass.
2. `./frontend.sh && yarn run cypress run` should pass locally.
